pymodbus/transport/transport.py

```diff
@@ -188,14 +192,17 @@ def __init__(
             self.comm_params.comm_type = CommType.TCP
             parts = host.split(":")
             host, port = parts[1][2:], int(parts[2])
-        self.init_setup_connect_listen(host, port)
 
-    def init_setup_connect_listen(self, host: str, port: int) -> None:
+        rs485_settings = self.comm_params.rs485_settings
```

Comment:
something if not correct here, in client/base.py you create a local variable, and do not use comm_params.

Reply:
No local variable created in client/base.py - I don't understand